docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+```
+
+### Populating the Template File
+When we populate this file we make reference to the available methods within the deployed smart contract **assetTransfer.js** file that can be found in: **fabric-samples** -> **asset-transfer-basic** -> **chaincode-javascript** -> **lib** -> **assetTransfer.js**
+
+#### Populate initializeWorkloadModule
+This method is used to prepare any items required by the primary `submitTransaction` method as the benchmark completes. 
+
+The number of assets to be created will be given as `roundArguments.assets`. We create assets using the smart contract by populating an arguments object, which defines the transaction body, and using the Caliper API `sendRequests`, which requires knowledge of:
+- contractId, the name of smart contract that is to be used and is present within the Caliper network configuration file
+- contractFunction, the specific function within the smart contract to invoke
+- contractArguments, the arguments to pass to the smart contract function
+- invokerIdentity, the identity to use that is present within the Caliper network configuration file
+- readOnly, if performing a query operation or not
+
+The method should look like this:
+```js
+    async initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext) {
+        await super.initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext);
+
+        for (let i=0; i<this.roundArguments.assets; i++) {
+            const assetID = `${this.workerIndex}_${i}`;
+            console.log(`Worker ${this.workerIndex}: Creating asset ${assetID}`);
+            const request = {
+                contractId: this.roundArguments.contractId,
+                contractFunction: 'CreateAsset',
+                invokerIdentity: 'Admin@org1.example.com',
+                contractArguments: [assetID,'blue',20,'penguin',500],
+                readOnly: false
+            };
+
+            await this.sutAdapter.sendRequests(request);
+        }
+    }
+```
+
+In the above example, different assets will be created that have the same parameters (blue, 20, penguin, 500). Comparing the above to the smart contract method itself, it should be evident that there is a 1:1 mapping of contract arguments to the method parameters.
+
+#### Populate submitTransaction
+This method runs repeatedly in the benchmark test phase. We will be evaluating the `ReadAsset` smart contract method by querying the assets we created in the `initializeWorkloadModule` method. 
+
+First, create a string identity for the asset to query, formed by the concatenation of the worker index and a random integer between 0 and the number of created assets. 
+
+Then await the call on `sendRequests`, passing the contractId, contractFunction` set as `queryCar`; `invokerIdentity` set as `admin@org1.example.com`; and `chaincodeArguments` set as an array that contains the asset to query in this run. 
```

Comment:
Something has gone wrong with the quoting on this line, plus `queryCar` isn't the correct contractFunction now

Reply:
yeah, looks like i missed a backtick - modified the paragraph to try and make it more clear